include/smpp34pdu_tlv_macros.hrl

```diff
@@ -5,9 +5,15 @@
 -define(TLV_UNPACK_UNEXPECTED(), unpack_tlv_fields(<<Unexpected:?TLV_TAG_SIZE, _/binary>>=Bin, Body) ->
     {_, Rest} = tlv:unpack(Unexpected, Bin),
     unpack_tlv_fields(Rest, Body)).
+-define(TLV_UNPACK_UNEXPECTED(RECORD_NAME), unpack_tlv_fields(<<Unexpected:?TLV_TAG_SIZE, _/binary>>=Bin, Body) ->
+    case tlv:unpack(Unexpected, Bin) of
+        {<<>>, Rest} -> unpack_tlv_fields(Rest, Body);
+        {Val, Rest} ->
+            #RECORD_NAME{vendor_specific = Vendor} = Body,
```

Comment:
vendor specific TLV keys reserved from 0x1400 to 0x3FFF

Reply:
Fixed